tools/toolchains/mbed_toolchain.py

```diff
@@ -1393,3 +1393,29 @@ def report(self):
         to_ret['linker'] = {'flags': copy(self.flags['ld'])}
         to_ret.update(self.config.report)
         return to_ret
+
+def replace_small_c_lib(target, toolchain):
+    """
+    Check if the small C lib should be replaced with the standard C lib.
+    Return True if the replacement occurs otherwise return None.
+    """
+    if (
```

Comment:
this should be 'return <condition>', not if. In most other languages this would say that the function was missing 'return False'. Although this is the python default, I wouldn't rely on it.

Reply:
c57d367